packages/components/src/select-control/index.tsx

```diff
@@ -89,15 +115,17 @@ function SelectControl(
 				label={ label }
 				size={ size }
 				suffix={
-					<DownArrowWrapper>
-						<Icon icon={ chevronDown } size={ 18 } />
-					</DownArrowWrapper>
+					props.suffix || (
+						<DownArrowWrapper>
+							<Icon icon={ chevronDown } size={ 18 } />
+						</DownArrowWrapper>
+					)
 				}
+				prefix={ props.prefix }
 				labelPosition={ labelPosition }
-				{ ...props }
 			>
 				<Select
-					{ ...props }
+					{ ...omit( props, 'prefix', 'suffix' ) }
```

Comment:
Instead of omitting `prefix` and `suffix`, could we just destructure them at the top of the render function?